CHANGES.rst

```diff
@@ -0,0 +1,58 @@
+Version 0.7 (unreleased)
```

Comment:
We follow semver and I don't think we will have two releases "on the go" in parallel for quite a while. This means that we will keep releasing 0.6.x until we have changes that require us to bump to 0.7.0. And then there won't be any more 0.6.x+1 releases ever again.

So I'd write `Version 0.x (unreleased)` here to have it all together. I think this will make it easy to put "Version 0.6.13" at short notice and release it when we are about to merge a PR that will move us to 0.7. Instead of having to have that PR hang around a long time. WDYT?